yarn.lock

```diff
@@ -2254,9 +2268,9 @@ electron-window-state@^5.0.3:
     mkdirp "^0.5.1"
 
 electron@^17:
-  version "17.4.2"
-  resolved "https://registry.yarnpkg.com/electron/-/electron-17.4.2.tgz#fb5226100fc3f5d6e319ccad7fa462bf15ee0d94"
-  integrity sha512-SEf0Tsc1GjCccdsmrEtT51DeaF48ZUa711ekVCmVczn40WvKaqQu7M0SB3c/qRLRQyek/Qyi56zpcoOkjJ8L0Q==
+  version "17.4.4"
```

Comment:
This version doesn't seem to exist 
<img width="972" alt="image" src="https://user-images.githubusercontent.com/2403652/169051842-fe1d1248-dac7-4a1c-9edc-939b0af2d1b9.png">
Hence the 403 when downloading `https://atom.io/download/electron/v17.4.4/node-v17.4.4-headers.tar.gz`

Though is a GH release https://github.com/electron/electron/releases/tag/v17.4.4